README.md

```diff
@@ -26,10 +26,15 @@ on:
       - '*.*.*'
   workflow_dispatch:
 
+permissions: {}
+
 jobs:
   auto-tag:
     name: Auto-tag
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
This list I'm referencing in https://github.com/silverstripe/silverstripe-framework/issues/10510#issuecomment-2055406330 only mentions contents?

Reply:
Yup contents only is correct - that what we did here https://github.com/silverstripe/gha-action-ci/blob/1/.github/workflows/auto-tag.yml#L15

Updated